Lib/test/support/ast_helper.py

```diff
@@ -41,3 +41,19 @@ def traverse_compare(a, b, missing=object()):
             elif a != b:
                 self.fail(f"{a!r} != {b!r}")
         traverse_compare(ast1, ast2)
+
+def to_tuple(t):
```

Comment:
Would it be possible to add it to `Lib/test/test_ast/__init__.py` or (new file) Lib/test/test_ast/utils.py?